.travis.yml

```diff
@@ -6,16 +6,31 @@ php:
     - 5.5
     - 5.6
     - hhvm
+    - 7.0
 
 env:
     - WP_VERSION=latest WP_MULTISITE=0
     - WP_VERSION=latest WP_MULTISITE=1
-    - WP_VERSION=4.1 WP_MULTISITE=0
-    - WP_VERSION=4.1 WP_MULTISITE=1
-    - WP_VERSION=4.0 WP_MULTISITE=0
-    - WP_VERSION=4.0 WP_MULTISITE=1
-    - WP_VERSION=3.9 WP_MULTISITE=0
-    - WP_VERSION=3.9 WP_MULTISITE=1
+    # Current stable
+    - WP_VERSION=4.5 WP_MULTISITE=0
```

Comment:
Agree with you on avoiding duplication. Bumping the current stable to 4.4.2 and previous stable to 4.4.1. Keeping latest as is.